src/sentry/api/event_search.py

```diff
@@ -222,10 +222,18 @@ def visit_search_term(self, node, children):
         return search_term[0]
 
     def visit_raw_search(self, node, children):
+        value = node.text
+
+        while value.startswith('"') and value.endswith('"') and value != '"':
```

Comment:
I was thinking we just needed to drop off the first set of quotes rather than remove all sets. Not sure it matters too much either way?